packages/shared/lib/features/interfaces/onboarding-features-for-network.interface.ts

```diff
@@ -0,0 +1,18 @@
+import { IFeatureFlag } from './feature-flag.interface'
+
+export interface IOnboardingFeaturesForNetwork extends IFeatureFlag {
```

Comment:
`INetworkOnboardingFeatures` is better IMO, slightly less verbose

Reply:
But it's not the network onboarding features it's the onboarding features for each network